webapp/javascript/components/TimelineChart/Annotations.plugin.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import * as ReactDOM from 'react-dom';
+import Color from 'color';
+import { Provider } from 'react-redux';
+import store from '@webapp/redux/store';
+import { CtxType } from './types';
+import extractRange from './extractRange';
+import AnnotationMark from './AnnotationMark';
+
+type AnnotationType = {
+  content: string;
+  timestamp: number;
+  type: 'message';
+  color: Color;
+};
+
+interface IFlotOptions extends jquery.flot.plotOptions {
+  annotations?: AnnotationType[];
+  wrapperId?: string;
+}
+
+(function ($) {
+  function init(plot: jquery.flot.plot & jquery.flot.plotOptions) {
+    plot.hooks!.draw!.push((plot, ctx: CtxType) => {
```

Comment:
```suggestion
    plot.hooks!.draw!.push((plot, ctx: CanvasRenderingContext2D) => {
```
I don't think we need custom types here.

Reply:
nice, removed my custom `CtxType`